File: Data/Models/Usuario.cs
namespace TrabalhoAPI.Data.Models
{
    public class Usuario
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Login { get; set; }
        public string Senha { get; set; }
        public List<Watchlist>? Watchlists { get; set; }
        public List<GeneroUsuario>? GeneroUsuarios { get; set; }
        public List<int>? FilmeIds { get; set; }
    }
}

File: src/app/Usuario.ts
import { GeneroUsuario } from "./GeneroUsuario";
import { Watchlist } from "./Watchlist";

export class Usuario{
    id: number = 0;
    nome: string = '';
    login: string = '';
    senha: string = '';
    watchlists: Watchlist[] = [];
    generoUsuarios: GeneroUsuario[] = [];
    filmeIds: number[] = [];
}